ObjObsSAP.tex

```diff
@@ -355,26 +354,26 @@ \subsubsection{Non-compulsory Parameters}
 server side. These parameters should be treated as reserved keywords.
 
 \begin{itemize}
-\item{\textbf{VIS\_MIN}\\A service \textbf{MAY} have a search parameter
-called \textbf{VIS\_MIN}. This parameter would constrain the visibility
-check to those time periods with at least the minimum visibility specified
-in the parameter. The unit of \textbf{VIS\_MIN} parameters must be expressed
+\item{\textbf{MIN\_OBS}\\A service \textbf{MAY} have a search parameter
```

Comment:
`min_obs` seems to be the new name for two columns in the former version: `min_viz` (l.259 in old version) and `viz_min` (l.358 in old version). Is it normal?

Reply:
Yes, it is correct. 
We wanted to clarify that we are no longer using the term visibility in any part of the document. That’s why we changed vis_min to min_obs.